#!/usr/bin/env python
PACKAGE = "vizzy_navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("k_ro",    double_t,    0, "Distance gain", 3, 0,   50)
gen.add("k_alpha", double_t, 0, "Orientation to goal gain", 8, 0,   50)
gen.add("k_beta",    double_t,    0, "Desired orientation gain", -1.5, -20,   20)


exit(gen.generate(PACKAGE, "vizzy_navigation", "Docking"))